sequelize/db.js

```diff
@@ -0,0 +1,58 @@
+const { Sequelize, DataTypes } = require('sequelize')
+
+let sequelize
+const dbModule = {}
+
+dbModule.start = ()=>{
+  sequelize = new Sequelize(`${process.env.DATABASE}`, `${process.env.USERNAME}`, `${process.env.PASSWORD}`, {
+    host: process.env.HOST || 'localhost', 
+    dialect: process.env.DIALECT || 'postgres',
+    pool: {
+      max: 5,
+      min: 0,
+      idle: 10000
+    }
+  })
+  return sequelize.authenticate()
+}
+
+dbModule.Account = ()=>{
+  return sequelize.define('account', {
```

Comment:
We will not be defining Account other than the time when sequelize has started, so this function is not necessary. Should be added to `start` function.